tests/tests/tier0/hirte-agent-started-and-connected/test_agent_started_and_connected.py

```diff
@@ -4,18 +4,23 @@
 import pytest
 import socket
 
+from podman.domain.containers import Container
+
 from hirte_tests.gather import journal
 from hirte_tests.provision import containers
 
+
 @pytest.fixture(autouse=True)
 def handle_env(
-    hirte_controller_ctr,
-    hirte_node_foo_ctr,
-    tmt_test_data_dir,
+    hirte_controller_ctr: Container,
+    hirte_node_foo_ctr: Container,
+    tmt_test_data_dir: Container,
 ):
     # initialize configuration
-    containers.put_files(hirte_controller_ctr, 'controller.conf.d', '/etc/hirte/hirte.conf.d')
-    containers.put_files(hirte_node_foo_ctr, 'node-foo.conf.d', '/etc/hirte/agent.conf.d')
+    containers.put_files(hirte_controller_ctr,
+                         'controller.conf.d', '/etc/hirte/hirte.conf.d')
```

Comment:
If you want to split the line, wouldn't it be better to have one parameter per line?

Reply:
This is due to the auto-formatter `autopep8` I chose - it just due to the line length. Not sure if there is an option for that.

Reply:
OK